### ISSUE TITLE
LazyDragSelectVerticalGrid not available for Android?

### ISSUE THREAD

**Issue by StefanOltmann** [jordond/drag-select-compose#45]
My dependencies for the compose module look like this:

```
        val commonMain by getting {

            dependencies {

                // Shared lib
                api(project(":shared"))

                // Compose Multiplatform
                api(compose.runtime)
                api(compose.foundation)
                api(compose.material)

                api("com.dragselectcompose:dragselect:2.1.2")
            }
        }
```

For JVM it works, but on Android it does not compile:
`Unresolved reference: LazyDragSelectVerticalGrid`

**Comment by jordond** [comment]
Thank you for reporting, I'll look into it 👍

**Comment by jordond** [comment]
I've kicked off a new release, attempted fix should be available in 2.1.3

**Comment by jordond** [comment]
Fixed in 2.1.3
